llvm/utils/update_cc_test_checks.py

```diff
@@ -173,7 +173,7 @@ def config():
     )
     parser.add_argument("--llvm-bin", help="llvm $prefix/bin path")
     parser.add_argument(
-        "--clang", help='"clang" executable, defaults to $llvm_bin/clang'
+        "--cc", "--clang", help='"clang" executable, defaults to $llvm_bin/clang'
```

Comment:
This changes the way the argument is accessed: it is now args.cc instead of args.clang so will break all tests.